alternator/rjson.hh

```diff
@@ -80,6 +80,18 @@ using string_buffer = rapidjson::GenericStringBuffer<encoding>;
 using writer = rapidjson::Writer<string_buffer, encoding>;
 using type = rapidjson::Type;
 
+extern allocator the_allocator;
```

Comment:
Is this pull request based on some other one? If so, what happened to that other one? I vaguely remember suggesting to wrap this allocator in `internal` namespace and you agreeing, but I don't see any trace of this discussion anywhere in here.

Reply:
Last one was an RFC. But same series, comments dealt with an implementation complete and tested. But I apparently managed to drop that change when rebasing. I suck. Will fix.